### ISSUE TITLE
Fine-tune model (maybe Stable Diffusion v1.5) on custom dataset

### ISSUE THREAD

**Issue by Parth-Sahastrabuddhe** [Parth-Sahastrabuddhe/Deep-Learning-Project#9]

**Comment by Parth-Sahastrabuddhe** [comment]
Implemented Stable diffusion on Celeb dataset.
Accuracy was high but the model was not learning as the loss did not change.

**Comment by Parth-Sahastrabuddhe** [comment]
Adding Early stopping and learning rate scheduler to get better results.
